docs/codegen-tour.md

```diff
@@ -687,6 +688,26 @@ fancy `RecordData` struct that would be a thin wrapper around a record plus the
 parent data, and which would implement the record getters, but deref to the
 record otherwise.... I'm really not sure.
 
+### <a id="arrays"></a> arrays
+
+Arrays of scalars, offsets, and records occur frequently in the spec. How we
+represent these arrays in codegen depends on what we know about the size of the
+contents the array. There are essentially three cases:
```

Comment:
This feels like it could be terser.

How you declare an array varies based on the nature of it's elements: